web/book/src/tutorial/relations.md

```diff
@@ -1,78 +1,110 @@
-basic query would be:
+PRQL is designed on top of _relational algebra_, which is the established data
+model used by modern SQL databases.
+A _relation_ has a rigid mathematical definition,
+which can be simplified to "a table of data".
+For example, the `invoices` table from the Chinook database[^1] looks like this:
+
+| invoice_id | customer_id | billing_city | _other columns_ | total |
+| ---------- | ------------ | ------------ | :-----------: | ----- |
+| 1        |  2 | Stuttgart | ...  | 1.98 |
+| 2        |  4 | Oslo      | ...        | 3.96 |
+| 3        |  8 | Brussels  | ...        | 5.94 |
+| 4        | 14 | Edmonton  | ...         | 8.91 |
+| 5        | 23 | Boston    | ...         | 13.86 |
+| 6        | 37 | Frankfurt | ...         | 0.99 |
+
+A table (relation) is composed of columns, each of which has an unique name and a designated data type.
+Every table has zero to many rows, each containing the same set of columns.
+The table above has `invoice_id` and `customer_id` columns with a data type of "integer number",
+a `billing_city` column with a data type of "text",
+a number of other columns, and
+and a `total` column that contains floating-point number. [^2]
+
+## Queries
+
+PRQL is designed to build queries that combine and select data from relations such as the `invoices` table above. Here is the most basic query:
 
 ```
-from albums
+from invoices
 ```
 
-This is not impressive, it is just the same table as we had before. But we can
-add on to it:
+_Note: You can try each of these examples here in the [Playground.](https://prql-lang.org/playground/)
+Enter the query on the left-hand side,
+and click **output.arrow** in the right-hand side to see the result._
+
+The result of the query above is not terribly interesting, it's just the same table as before.
+But PRQL can do more...
+
+**derive:** PRQL can also _add_ columns to a table with the `derive` statement.
+Let's define a new column for Value Added Tax, set at 19% of the invoice total.
+_(Try it in the [Playground.](https://prql-lang.org/playground/))_
 
 ```
-from albums
-derive artist_id + 1
+from invoices
+derive { total * 0.19 }
 ```
 
 <!-- todo: make sure that the new column is unnamed -->
 
-`derive` adds a new column to the relation, computed from other columns. Note
-that the new column is unnamed. If we wanted to have it named, we could say
-something like:
+The value of the new column can be a constant (such as a number or a string),
+or can be computed from the value of an existing column.
+Note that the name of the new column is the expression that defined it.
+To give that column an interesting name,
+assign the expression to a name `value_added_tax`.
 
 ```
-from albums
-derive my_column = artist_id + 1
+from invoices
+derive { value_added_tax = total * 0.19 }
 ```
 
-Now let's say that we don't need the original `artist_id` column and would want
-to remove it. We can use select:
+**select:** PRQL can also _remove_ columns from a table.
```

Comment:
```suggestion
**select:** PRQL can also select specific columns from a table.
```

I don't really see `select` as removing, it's more like you're narrowing to the things you want. But I realize `select` is in the definition, which isn't great